src/scss/components/_filter-button.scss

```diff
@@ -0,0 +1,21 @@
+@use 'abstracts' as *;
+
+.filter__button {
+  .icon {
+    @include font-size(12px);
+  }
+
+  &:focus:hover {
+    box-shadow: 0 0 0 2px #85b7db;
+  }
+}
+
+.filter__reset {
+  &:focus {
+    box-shadow: none;
```

Comment:
The reset button should have a box-shadow on focus. Right now, there's no way to tell if the reset button is focused when navigating via a keyboard.